tools/docker/start.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+sudo sh -c 'echo "
```

Comment:
@gsnedders wrote something to generate this output, can you use that here?

Reply:
`./wpt make-hosts-file` does this